core/domain/exp_jobs_one_off.py

```diff
@@ -363,3 +363,60 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         yield (key, values)
+
+
+class FallbacksToHintsOneOffJob(jobs.BaseMapReduceJobManager):
+    """Job that converts existing Fallbacks to Hints for explorations with
+    state schema version 10.
+
+    IMPORTANT: The GAE memcache must be flushed after running this job.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+        if item.states_schema_version != 10:
```

Comment:
Hi @seanlip -- This job is to convert Fallbacks to Hints for explorations that are already in the latest states schema version. 

I've added this logic in the appropriate conversion function, but that gets executed only for explorations that do not have the latest states schema version. Explorations that are currently in the latest version don't undergo this migration -- these are explorations that were created since the back-end PR (#3533) was merged. If creators made new explorations during this period and specified fallbacks in them, those fallbacks won't be converted. That said, I don't think job is absolutely necessary because:
1. I don't think there will be too many explorations that might fall in this category (maybe even none).
2. If we're emailing creators regarding this conversion, this conversion doesn't seem necessary.

Also, in this PR, I've modified the update_interaction_fallbacks function to copy new fallbacks to hints as they are created/updated. So this will handle conversions of newly added fallbacks, until Hints UI is introduced.